src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -52,7 +53,7 @@
                 <div class="col-sm-offset-2 col-md-offset-3 col-sm-8 col-md-6">
                   <div class="card-pf card-pf-aggregate-status">
```

Comment:
please remove the gray background that is present - according to the inspector, it is coming from the Spaces component scss file.